src/ngraph/op/convert.hpp

```diff
@@ -53,6 +54,10 @@ namespace ngraph
                     m_destination_type = destination_type;
                 }
 
+                size_t get_version() const override { return 0; }
```

Comment:
Get rid of this. `Node::get_version()` gets the version from the type_info.

Reply:
done